RELEASE.md

```diff
@@ -303,8 +301,14 @@ After these steps are completed, the following should occur:
     - `quay.io/bpfman/bpfman-operator`
     - `quay.io/bpfman/bpfman-agent`
 
-After the release completes:
+After the release is complete do the following:
 
+- Run `make build-release-yamls` from the `bpfman-operator/` directory, and then
```

Comment:
Good catch!  Breaking out this step was missed after the repo split.

Reply:
Yes, I looked back at the assets and haven't been there since the repo split.